C#/API/Mat.cs

```diff
@@ -565,53 +565,73 @@ public double[] ToTxyzRxyz()
 		/// <returns>XYZWPR translation and rotation in mm and radians</returns>
```

Comment:
It's not part of the bug, but while modifying this method it might be a good time to fix the docs: the result doesn't include Euler angles, it has a rotation vector (and similarly, it doesn't make sense to describe the return value as XYZWPR)

Reply:
It's done, thank you